services/distribution/src/test/resources/application.yaml

```diff
@@ -197,7 +198,8 @@ services:
         base-url: ${BUSSINESS_RULES_BASE_PATH:https://distribution-dfe4f5c711db.dcc-rules.de/}
         country-list-path: ${BUSSINESS_RULES_COUNTRY_LIST_PATH:/countrylist}
         value-sets-path: ${BUSSINESS_RULES_VALUE_SETS_PATH:/valuesets}
-        rules-path: ${BUSSINESS_RULES_RULES_PATH:/rules}
+        rules-path: ${BUSSINESS_RULES_PATH:/rules}
```

Comment:
In the src/application.yml a lot of changes were made but not so many in the test/application.yaml, are there any missing here?

Reply:
I will adjust the main application.yaml